src/components/PoolingWidget/index.tsx

```diff
@@ -166,10 +173,11 @@ const PoolingInterface: React.FC = () => {
             setTxHash(txHash)
 
             batchedUpdates(() => {
+              let orderId = lastOrderIdRef.current + 1
               orders.forEach(({ buyToken: buyTokenId, sellToken: sellTokenId, buyAmount, sellAmount }) => {
                 const pendingOrder = {
                   txHash,
-                  id: 'PENDING ORDER',
+                  id: (orderId++).toString(10),
```

Comment:
I think it was wrong to have `'PENDING ORDER'` as the ID an I see the point on guessing it from our current orders, but in general I think it would be fine to create a unique temporal ID (i.e. using a timestamp) and then DELETE the pending order when it's confirmed.

I remember working in this part of the code, and it's asking for a bit of refactor. Looks a bit complex right now, and we keep adding more complexity.

Reply:
Prending orders are being deleted anyway. Once tx is mined